torch_np/_ufuncs.py

```diff
@@ -76,10 +80,14 @@ def matmul(
         raise NotImplementedError
 
     # NB: do not broadcast input tensors against the out=... array
-    result = _binary_ufuncs.matmul(*tensors)
+    result = _binary_ufuncs_impl.matmul(*tensors)
     return result
 
 
+matmul.__qualname__ = "matmul"
+matmul.__name__ = "matmul"
```

Comment:
These, as they come from `normalizer` already have the right name and qualname. ditto for th eother one-offs.

Reply:
Removed and force-pushed.